packages/node/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "public": true,
   "name": "@module-federation/node",
-  "version": "0.3.0",
+  "version": "0.1.0",
```

Comment:
@ScriptedAlchemy probably an unintentional version change

Reply:
My semver tooling seems to be totally broken. Having to do manual releasing